.github/workflows/swiftshader.yml

```diff
@@ -25,15 +25,16 @@ jobs:
 
     - name: Build Filament with SwiftShader Support
       run: |
-          docker run -v `pwd`:/trees/branch ssfilament branch/build.sh -t release
+          docker run -v `pwd`:/trees/branch ssfilament branch/build.sh -t debug
```

Comment:
Do we need to build everything? Could we just build gltf_viewer to reduce turnaround times?

Reply:
That's a good idea, will do.  (The last few commit were tests btw, I will remove them.)